packages/babel-plugin-bugfix-firefox-class-in-computed-class-key/test/fixtures/basic/yield-await/output.mjs

```diff
@@ -0,0 +1,21 @@
+class A {
+  #x;
+  [(() => class {}).call()];
+  [await (async () => await class {})()];
+}
+async function* f() {
+  class A {
+    #x;
+    [(() => class {}).call()];
```

Comment:
We need to add `.call()` here to prevent Terser from undoing the work done by this plugin: https://github.com/terser/terser/issues/1514

Reply:
I think this is fine, because it is possible that more than just terser will do this (or may do so in the future).